packages/deprecated/src/index.js

```diff
@@ -30,6 +38,20 @@ export default function deprecated( feature, { version, alternative, plugin, lin
 		return;
 	}
 
+	/**
+	 * Fires whenever a deprecated feature is encountered
+	 *
+	 * @param {string}  feature             Name of the deprecated feature.
+	 * @param {?Object} options             Personalisation options
```

Comment:
I'm wondering if there is a way to turn `options` into shape to avoid duplication. I saw something like this in the past:
https://github.com/WordPress/gutenberg/blob/master/packages/components/src/autocomplete/index.js#L93-L105

Reply:
Yes, we didn't come up with any standards around that. It might make sense to follow PHP implementation though.